corehq/messaging/scheduling/tests/test_recipients.py

```diff
@@ -638,7 +680,9 @@ def test_expand_group_recipients(self):
         )
 
     def test_mobile_worker_recipients_with_user_data_filter(self):
-        schedule = TimedSchedule.create_simple_daily_schedule(
+        schedule = self._create_schedule(user_data_filter={'role': ['nurse']})
+
+        TimedSchedule.create_simple_daily_schedule(
```

Comment:
Does this `TimedSchedule` still need to be created?

Reply:
It does not. Thanks for catching that.